src/styles/index.ts

```diff
@@ -2,3 +2,5 @@ import "@fontsource/roboto/300.css";
 import "@fontsource/roboto/400.css";
 import "@fontsource/roboto/500.css";
 import "@fontsource/roboto/700.css";
```

Comment:
Is design system use `Roboto` font?

If not, create an `config/fonts.ts` file to declare the custom font in that file